website/pages/docs/concepts/policies.mdx

```diff
@@ -173,8 +173,22 @@ capabilities, which controls a token's access to credentials in Vault.
 ~> Policy paths are matched using the **most specific path match**. This may be
 an exact match or the longest-prefix match of a glob. This means if you define a
```

Comment:
Like my other comment, once we get consensus on a replacement for "longest-prefix match" we should put it here as well.